tests/test_fx/test_pipeline/test_timm_model/timm_utils.py

```diff
@@ -0,0 +1,51 @@
+import torch
+from torch.fx import symbolic_trace
+from torch.fx import GraphModule
+from colossalai.fx.passes.adding_split_node_pass import split_with_split_nodes_pass, balanced_split_pass
+from colossalai.fx import ColoTracer
+import inspect
+import random
+import numpy as np
+
+MANUAL_SEED = 0
+random.seed(MANUAL_SEED)
+np.random.seed(MANUAL_SEED)
+torch.manual_seed(MANUAL_SEED)
+torch.backends.cudnn.deterministic = True
+
+
+def split_model_and_compare_output(model, data, meta_args=None):
+    model.eval()
```

Comment:
I am not sure why timm tests will fail for control-flow-based models. You can try to debug this by [adding hooks to modules](https://pytorch.org/docs/stable/generated/torch.nn.modules.module.register_module_forward_hook.html) to check which modules produce inconsistent outputs.